### ISSUE TITLE
Promise not resolved for fetch

### ISSUE THREAD

**Issue by tiabc** [status-im/status-go#231]
@cristian.sandu on our Slack channel:
> ok, so about fetch() I see the call get on the http server and gets an 200 answer but the promise does not seem to be resolved
```go
try {
       fetch(‘http://10.162.130.120:7878/pending.json’).then(function (data) {
           result.data = data;
           status.sendMessage(‘Done!‘);
           console.log(‘done’);
       }).catch(function (e) {
           result.err = e;
           status.sendMessage(‘Failed!‘);
           console.log(‘err’);
       });
   } catch (e) {
       result.err = e;
       status.sendMessage(‘Error!‘);
       console.log(‘error’);
   }
```
> there is no error but no response either

**Comment by influx6** [comment]
The issue for this is related back to #208, has fetch also depends on the event loop. I need to investigate `ottoext` to come up with a strategy to call `ottoext.Loop.Run()`, my current approach of taking the previous `fetch` handle and map it with one that calls the needed method fails. 

Will dive into `ottoext/fetch` package then come up with a solution for this.

**Comment by tiabc** [comment]
Fixed in #293
